modules/ROOT/pages/mule-object-stores.adoc

```diff
@@ -0,0 +1,89 @@
+= Store Application Data Using Object Stores
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, object, store
+
+An object store is a facility for storing objects in or across Mule applications. Mule runtime engine(Mule) uses object stores to persist data for eventual retrieval. Internally, Mule uses object stores in various filters, routers, and other message processors that need to store states between messages.
+
+If you deploy your application to CloudHub, you can also use Object Store V2.
+
+== Use Cases
+
+You can configure a custom object store to use in the following scenarios:
+
+* When configuring an Idempotent Message Validator, to store the unique IDs in the specified object store.
+* When configuring a Redelivery policy in a listener(HTTP, File, JMS, VM, Socket), to store the redelivery count in the specified object store.
+* When configuring a Cache Scope that uses a custom caching strategy, to store the cached data in the specified object store.
```

Comment:
I tried to look up Cache Scope and got only "cache scope" and "Cache scope." 
Since you precede it with "a", I think we want "a" generic thing, not "the" named thing, and so I suggest lowercase.
Please let me know if this is wrong, for the Term Presentation List?
```suggestion
* When configuring a cache scope that uses a custom caching strategy, to store the cached data in the specified object store.
```

Reply:
https://docs.mulesoft.com/mule-runtime/4.3/cache-scope "Cache scope"